easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,74 @@
+# by Christian Meesters, Johannes Gutenberg-University Mainz
+
+easyblock = "CmdCp"
+
+name = "rmats-turbo"
+version = "4.1.1"
+
+homepage = "https://github.com/Xinglab/rmats-turbo"
+description = """
+            rMATS turbo is the C/Cython version of rMATS
+            (refer to http://rnaseq-mats.sourceforge.net). The major
+            difference between rMATS turbo and rMATS is speed and space
+            usage. rMATS turbo is 100 times faster and the output file is
+            1000 times smaller than rMATS. These advantages make analysis
+            and storage of a large scale dataset easy and convenient.
+              """
+
+toolchain = {"name": "GCCcore", "version": "10.2.0"}
+
+source_urls = ["https://gitlab.com/Xinglab/archive/refs/tags/"]
```

Comment:
Any special reason to use github for homepage, but sources from gitlab? In fact, this path doesn't really look right at all?